### ISSUE TITLE
Close button X in top right unresponsive

### ISSUE THREAD

**Issue by alleyoopster** [owncloud/documents#517]
### Steps to reproduce
1. Open document to work in
2. Finish up and click on X in top right

### Expected behaviour
Document should save and close

### Actual behaviour
Nothing
Did once get error
```
ASSERTION FAILED:
tried to unsubscribe unknown callback from event "input/compositionstart"
```

### Server configuration
**Operating system**: Linux

**Web server:** Nginx

**Database:** mysql

**PHP version:** 5.6.9-0

**ownCloud version:** 8.1

**Updated from an older ownCloud or fresh install:** update

**List of activated apps:**

```Enabled:
 - activity: 2.0.1
 - contacts: 0.4.0.0
 - external: 1.2
 - files: 1.1.9
 - files_external: 0.2.3
 - files_locking: 
 - files_mv: 0.8.1a
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.2
 - files_texteditor: 0.4
 - files_trashbin: 0.6.3
 - files_versions: 1.0.6
 - files_videoviewer: 0.1.3
 - firstrunwizard: 1.1
 - gallery: 0.6.0
 - mail: 0.1.3
 - provisioning_api: 0.2
 - templateeditor: 0.1
Disabled:
 - bookmarks
 - calendar
 - calendar.old
 - calendarplus
 - direct_menu
 - documents
 - encryption
 - files_encryption
 - music
 - ownnote
 - roundcube
 - search_lucene
 - shorty
 - storagecharts2
 - tasks
 - tasksplus
 - user_external
 - user_ldap
 - user_webdavauth


```

**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => 'ocpt5ctcvmoi',
  'passwordsalt' => '<secret>',
  'secret' => '<secret>',
  'trusted_domains' => 
  array (
    0 => 'cloud.kusikiliza.com',
  ),
  'datadirectory' => '/data/owncloud',
  'overwrite.cli.url' => 'https://cloud.kusikiliza.com',
  'dbtype' => 'mysql',
  'version' => '8.1.0.8',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'oc_<secret>',
  'dbpassword' => '<secret>',
  'installed' => true,
  'forcessl' => true,
  'forceSSLforSubdomains' => true,
  'theme' => '',
  'maintenance' => false,
  'mail_from_address' => '<secret>',
  'mail_smtpmode' => 'smtp',
  'mail_domain' => 'gmail.com',
  'mail_smtphost' => 'smtp.gmail.com',
  'mail_smtpport' => '465',
  'mail_smtpsecure' => 'ssl',
  'mail_smtpauth' => 1,
  'mail_smtpname' => '<secret>',
  'mail_smtppassword' => '<secret>',
  'memcache.local' => '\\OC\\Memcache\\APCu',
);
```

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox 39.0

**Operating system:** Linux Mint 7.1 KDE

#### Web server error log
```
2015/08/01 09:42:59 [error] 27800#0: *11457 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 197.83.247.245, server: cloud.kusikiliza.com, request: "POST /index.php/apps/calendarplus/importeventscalendar HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock", host: "cloud.kusikiliza.com", referrer: "https://cloud.kusikiliza.com/index.php/apps/calendarplus/"
2015/08/01 09:52:14 [error] 27800#0: *11457 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 197.83.247.245, server: cloud.kusikiliza.com, request: "POST /index.php/apps/calendarplus/importeventscalendar HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock", host: "cloud.kusikiliza.com", referrer: "https://cloud.kusikiliza.com/index.php/apps/calendarplus/"
2015/08/01 10:17:15 [error] 27801#0: *12455 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 197.83.247.245, server: cloud.kusikiliza.com, request: "POST /index.php/apps/calendarplus/importeventscalendar HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock", host: "cloud.kusikiliza.com", referrer: "https://cloud.kusikiliza.com/index.php/apps/calendarplus/"
2015/08/01 18:41:23 [crit] 27802#0: *19909 SSL_do_handshake() failed (SSL: error:140A1175:SSL routines:SSL_BYTES_TO_CIPHER_LIST:inappropriate fallback) while SSL handshaking, client: 64.41.200.103, server: 0.0.0.0:443
2015/08/01 18:43:04 [crit] 27801#0: *20162 SSL_do_handshake() failed (SSL: error:14094085:SSL routines:SSL3_READ_BYTES:ccs received early) while SSL handshaking, client: 64.41.200.103, server: 0.0.0.0:443
2015/08/01 18:43:05 [crit] 27801#0: *20163 SSL_do_handshake() failed (SSL: error:14094085:SSL routines:SSL3_READ_BYTES:ccs received early) while SSL handshaking, client: 64.41.200.103, server: 0.0.0.0:443
2015/08/02 09:48:07 [error] 29020#0: *32306 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 62.210.250.116, server: cloud.kusikiliza.com, request: "GET /vtigercrm/vtigerservice.php HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "185.92.222.92"
2015/08/02 11:18:39 [error] 29018#0: *55129 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 197.87.145.61, server: cloud.kusikiliza.com, request: "PROPFIND /index.php HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "cloud.kusikiliza.com"
2015/08/02 12:36:39 [crit] 3765#0: *1 connect() to unix:/var/run/php5-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 197.87.145.61, server: cloud.kusikiliza.com, request: "PROPFIND /remote.php/caldav/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "cloud.kusikiliza.com"
2015/08/02 12:36:39 [crit] 3765#0: *1 connect() to unix:/var/run/php5-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 197.87.145.61, server: cloud.kusikiliza.com, request: "REPORT /remote.php/caldav/calendars/alleyoopster/contact_birthdays/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "cloud.kusikiliza.com"
2015/08/02 12:36:44 [error] 3765#0: *1 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 197.87.145.61, server: cloud.kusikiliza.com, request: "REPORT /remote.php/caldav/calendars/alleyoopster/birthdays/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "cloud.kusikiliza.com"
2015/08/02 12:36:44 [crit] 3765#0: *49 connect() to unix:/var/run/php5-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 197.87.145.61, server: cloud.kusikiliza.com, request: "REPORT /remote.php/caldav/calendars/alleyoopster/birthdays/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "cloud.kusikiliza.com"
2015/08/02 12:40:17 [crit] 3765#0: *257 connect() to unix:/var/run/php5-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 197.87.145.61, server: cloud.kusikiliza.com, request: "PROPFIND /index.php HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "cloud.kusikiliza.com"

```

#### ownCloud log (data/owncloud.log)
```
EMPTY 
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by VicDeo** [comment]
@alleyoopster mmm. In the first place I didn't get how it is possibe to edit a document when the app is disabled.

**Comment by alleyoopster** [comment]
@VicDeo Ooops. :) Very true, I must have made a change before or after I copied that list. It certainly was enabled when I had the problem.

**Comment by alleyoopster** [comment]
Sorry pressed the closed button by mistake

**Comment by VicDeo** [comment]
@alleyoopster which documents version and how it was installed? 
Is anything added to the log at the admin page after a document is being opened?

**Comment by alleyoopster** [comment]
Document version 0.10.0

No errors in the log after opening documents. I have been unable to 
reproduce this again lately.

**Comment by VicDeo** [comment]
@alleyoopster I recommend to update to 0.10.2 as there are a several heisenbugs fixed since 0.10.0 
It is published at the appstore a few hours ago.

**Comment by alleyoopster** [comment]
@VicDeo thanks, I've upgraded, I'll keep an eye on it. 

closing
